CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to Terms Types
+
+Thank you for considering contributing to the `terms-types` project!  🎉👍
```

Comment:
I don't think we can consider this a “project”. It is a component of Open Terms Archive 🙂